meson.build

```diff
@@ -45,6 +45,24 @@ if all_loaders or get_option('loaders').contains('jpg') == true
     config_h.set10('THORVG_JPG_LOADER_SUPPORT', true)
 endif
 
+if all_loaders or get_option('loaders').contains('webp') == true
```

Comment:
Unusual script here. Commens will be helpful.
"#We should check the dependency first because static webp is not supported."